src/electron/index.ts

```diff
@@ -219,8 +223,28 @@ app.on('ready', () => {
 
   // Set the app to launch at startup to connect automatically in case of a showdown while proxying.
   app.setLoginItemSettings({openAtLogin: true, args: [Options.AUTOSTART]});
+  if (isLinux && process.env.APPIMAGE) {
+    let AutoLaunch = require('auto-launch')
+    let outlineAutoLauncher = new AutoLaunch({
+      name: 'OutlineClient',
+      path: process.env.APPIMAGE,
+    });
+
+    outlineAutoLauncher.isEnabled()
+        .then(function(isEnabled: boolean) {
+          if (isEnabled) {
+            return;
+          }
+          outlineAutoLauncher.enable();
+        })
+        .catch((err: Error) => {
+          console.error('failed to add autolaunch entry for Outline');
+        });
+  }
 
-  if (process.argv.includes(Options.AUTOSTART)) {
+  // because autostart doesn't work for linux then we just assume we
```

Comment:
Not sure, I'd have to test. It may be fine if we rely on the presence of the connection configuration in storage, but I prefer to make it explicit. I don't know anything about `auto-launch`, but would it be possible to tell it to invoke the app with this option?
What is the benefit of excluding this check?